x-pack/plugins/security_solution/public/common/components/navigation/breadcrumbs/index.ts

```diff
@@ -110,8 +127,8 @@ export const getBreadcrumbsForRoute = (
       ),
     ];
   }
-  if (isAlertsRoutes(spyState) && object.navTabs) {
-    const tempNav: SearchNavTab = { urlKey: 'detections', isDetailPage: false };
+  if ((isAlertsRoutes(spyState) || isRulesRoutes(spyState)) && object.navTabs) {
```

Comment:
I am worried about that, do we need to do something different? do we also need to add `Exceptions`?

Reply:
Talked to @angorayc and she convinced me that we are in good shape here